README.md

```diff
@@ -4,30 +4,52 @@
 
-### References
-
 Journet6644: Journet, Catherine, et al. "Large-scale production of single-walled carbon nanotubes by the electric-arc technique." nature 388.6644 (1997): 756-758.
 
 Tulevski2007: Tulevski, George S., et al. "Chemically assisted directed assembly of carbon nanotubes for the fabrication of large-scale device arrays." Journal of the American Chemical Society 129.39 (2007): 11964-11968.
 
 Chen2014: Chen, Haitian, et al. "Large-scale complementary macroelectronics using hybrid integration of carbon nanotubes and IGZO thin-film transistors." Nature communications 5.1 (2014): 4097.
+```
+
+## What is PaperQA
+
+PaperQA is engineered to be the best RAG model for working with scientific papers. Specifically, it has the following features:
+
+- A simple interface to get good answers, with no hallucinations, grounding responses with in-text citations.
+- State-of-the-art implementation including metadata-awareness in document embeddings and LLM-based re-ranking and contextual summarization (RCS).
+- The ability to do agentic RAG which iteratively refines queries and answers.
+- Automatically obtained paper metadata, including citation and journal quality data.
+- A usable full-text search engine for a local repository of PDF/text files.
+- A robust interface for customization, with default support for all [LiteLLM](https://docs.litellm.ai/docs/providers) models.
+
+By default, it uses [OpenAI embeddings](https://platform.openai.com/docs/guides/embeddings) and [models](https://platform.openai.com/docs/models) with a numpy vector DB to embed and search documents. However, you can easily use other closed-source, open-source models or embeddings (see details below).
+
+PaperQA depends on some heroes that make our repo possible (in the order I remembered them):
+
+1. [Semantic Scholar](https://www.semanticscholar.org/)
+2. [Crossref](https://www.crossref.org/)
+3. [Unpaywall](https://unpaywall.org/)
+4. [Pydantic](https://docs.pydantic.dev/latest/)
+5. [Litellm](https://github.com/BerriAI/litellm)
+6. [pybtex](https://pybtex.org/)
+7. [pymupdf](https://pymupdf.readthedocs.io/en/latest/)
```

Comment:
Didn't we just remove this one?

Reply:
confusing - no we removed `pypdf` not `pymupdf`